src/client/helpers/validarCorreo.js

```diff
@@ -0,0 +1,21 @@
+export function validarEmail(valor) {
+  if (/^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3,4})+$/.test(valor)) {
+    alert('La dirección de email ' + valor + ' es correcta.');
```

Comment:
what's the purpose of this function? I would expect it to return `true` if the email is valid. false otherwise